tests/suites/test_suite_pk.data

```diff
@@ -710,35 +710,35 @@ pk_get_psa_attributes:MBEDTLS_PK_RSA:FROM_PAIR:PSA_KEY_USAGE_DECRYPT:1:PSA_ALG_R
 
 PSA attributes for pk: RSA v21 SHA-256 pair DECRYPT
 depends_on:PSA_WANT_KEY_TYPE_RSA_PUBLIC_KEY:PSA_WANT_ALG_RSA_PSS:PSA_WANT_ALG_SHA_256
-pk_rsa_v21_get_psa_attributes:MBEDTLS_MD_SHA256:FROM_PAIR:PSA_KEY_USAGE_DECRYPT:1:PSA_ALG_RSA_OAEP(PSA_ALG_SHA_256)
+pk_rsa_v21_get_psa_attributes:MBEDTLS_MD_SHA256:FROM_PAIR:PSA_KEY_USAGE_DECRYPT:1:PSA_ALG_RSA_PKCS1V15_CRYPT
```

Comment:
I'm not sure if there's any point in `pk_rsa_v21_get_psa_attributes`, since there isn't a concept of “v21 PK context” any more. But that can be part of a later cleanup. I don't want to get bogged down into a major refactoring of the test code.